### DISCUSSION TITLE
Multi-Language support

### DISCUSSION THREAD

**Discussion by ArtMin96** [filamentphp/filament#1380] [Help] [answered]
Thanks for a great job.

Does this admin panel have multiple language support? For example, I have several languages on the site and I want to fill in the title for all languages when creating something.

**Comment by danharrin** [comment, marked as answer]
Yes, check out our Translatable plugin in the docs.

**Reply by ArtMin96** [reply]
Thanks

**Reply by iboved** [reply]
@danharrin I just started using Filament. Could you please provide a link to the Translate plugin, and who is the author? Thanks!
